### PR TITLE
IN-1031 Apply the accordion pattern to current content

### PR DESCRIPTION
**Pull request by borodean**
Record on Airtable: [IN-1031](https://airtable.com/appEwS5Z32QZTF9fF/tblUiZfI5YQnjOcyy/viwYfw4fpn1IwMSqM/recrBUpyf9RpEzysb?blocks=hide)

Apart from sectioning the assignments according to @Patrickkaman, this PR also takes the opportunity to fix some formatting and spelling mistakes, and cleans up developer lists from duplicate people.

The track updates aren't pushed to prod yet.

### CODE REVIEW

File: container/01-creating-a-directory/assignment.md [hunk 1]
```diff
@@ -1,21 +1,29 @@
 ---
 slug: creating-a-directory
-id: 5y3ow5tpeafn
+id: sn24rtbabl3y
 type: challenge
 title: Creating a directory
 teaser: Learn how to create directory
+notes:
+- type: text
+  contents: Please wait while we set up the first challenge
```
Comment (borodean) [inline container/01-creating-a-directory/assignment.md:9]: This note is currently on prod, so I carried it over here as well.

**Review by victorfrederiknielsen — COMMENTED** [review]
I think there's an issue with changing of IDs. Perhaps the IDS are currently incorrect? Otherwise I believe the CD will fail.

File: aws-cloud-account/track.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 slug: aws-cloud-account
-id: 0ytrmq5i2xlu
+id: 44zxuqynye2j
```
[the remark this answers is not in the document]
Comment (borodean) [reply aws-cloud-account/track.yml]: b695bc63243a7af1e0201d56026231145aa4d28f

**Review by victorfrederiknielsen — APPROVED** [review]
🚀

**Pull request merged**